AutoCollection/Performance.ts

```diff
@@ -17,6 +17,8 @@ class AutoCollectPerformance {
     private _handle: NodeJS.Timer;
     private _isEnabled: boolean;
     private _isInitialized: boolean;
+    private _lastAppCpuUsage: { user: number, system: number } = {user: 0, system: 0};
```

Comment:
Nit: Because you did = here, you don't need to define the type explicitly. But also, it doesn't seem to be the pattern in this file to do assignment here.

Reply:
I also wonder if these should be statics. I haven't thought it through very much, but CPU usage isn't really going to change from class instance to class instance. Not that it matters very much because only one instance of this class should exist anyway..